### DISCUSSION TITLE
Wrong team assigned on faceit demo

### DISCUSSION THREAD

**Discussion by datamendes** [akiver/cs-demo-manager#954] [Q&A] [answered]
Hi,

I am currently collecting data from Faceit championships, and I've noticed that sometimes players are assigned to the wrong teams in the data. This leads to inaccurate information and affects the reliability of my analysis.

Can anyone help me how to fix this?

demo link:
https://www.faceit.com/en/cs2/room/1-a83383df-906d-419c-bf97-f3f325b99aeb

**Comment by akiver** [comment]
![Screenshot from 2024-10-22 07-58-11](https://github.com/user-attachments/assets/3a398d94-e7c5-4e52-ab56-cb92020f92a2)

![Screenshot from 2024-10-22 07-57-21](https://github.com/user-attachments/assets/deaae102-e4dc-4649-bc03-f0dcfa1804a8)

I don't see anything wrong?

**Reply by datamendes** [reply]
this is what i see when I analyze the demo.

I dont know why I got this result in my end.

![Screenshot 2024-10-22 133819](https://github.com/user-attachments/assets/2bf45886-05f9-43a3-ac34-bb79da74ec25)

**Reply by akiver** [reply]
what's the source of the demo? above the scoreboard

**Reply by datamendes** [reply]
Maybe thats the issue. 

I used this command to analyze the demos:

csdm xlsx "C:\Users\username\Desktop\MyFolder" --output-folder "C:\Users\username\Documents" --source esl

if I use:

csdm xlsx "C:\Users\username\Desktop\MyFolder" --output-folder "C:\Users\username\Documents" --source faceit

Should it fix the problem right?

![SS](https://github.com/user-attachments/assets/35251db4-a1a8-49e9-8cd6-7ba648d6f4e8)

**Reply by datamendes** [reply]
And with this command I can download all the info that makes the UI of CS demo manager right?

**Reply by akiver** [reply, marked as answer]
> csdm xlsx "C:\Users\username\Desktop\MyFolder" --output-folder "C:\Users\username\Documents" --source esl

you are analyzing a faceit demo with the ESL source, that's why you have incorrect scores.
you should use `--source` only if the source is not detected or incorrectly detected.

> And with this command I can download all the info that makes the UI of CS demo manager right?

the `xlsx` command generates XLSX files so it's more "export" than "download".
you can usage SQL for advanced usage, see the [doc](https://cs-demo-manager.com/docs/guides/export)

**Reply by datamendes** [reply]
got it thanks. problem solved
